Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -20,6 +20,11 @@ void ComputeBattleUnitAttack(struct BattleUnit * attacker, struct BattleUnit * d
     int status;
     status = GetItemMight(attacker->weapon);
 
+#if (defined(SID_LadyBlade) && (COMMON_SKILL_VALID(SID_LadyBlade)))
+    if (!(UNIT_CATTRIBUTES(&attacker->unit) & CA_FEMALE))
```

Comment:
also this results in male

Reply:
Ah true. I didn't noticed because Eirika had a Rapier and I was testing on Saar.